File: aurora-scala/src/main/scala/net/gree/aurora/scala/application/Repositories.scala
package net.gree.aurora.scala.application

import net.gree.aurora.scala.domain.datasource.DataSourceRepository
import net.gree.aurora.scala.domain.shardingconfig.ShardingConfigRepository

trait Repositories {

  val dataSourceRepository: DataSourceRepository

  val shardingConfigRepository: ShardingConfigRepository

}

File: aurora-core/src/main/java/net/gree/aurora/application/Repositories.java
package net.gree.aurora.application;

import net.gree.aurora.domain.datasource.DataSourceRepository;
import net.gree.aurora.domain.shardingconfig.ShardingConfigRepository;

public interface Repositories {

    DataSourceRepository getDataSourceRepository();

    ShardingConfigRepository getShardingConfigRepository();

}
